Add announcement post for OMERO 5.6.5 (#596)
ome/www.openmicroscopy.org: _posts/2022-06-29-omero-5-6-5.md

---
layout: post
title: Release of OMERO.server 5.6.5
---

Today we are releasing OMERO.server 5.6.5 which includes:

- a new server configuration allowing to control the pyramidal requirement for floating-point images
- an upgrade of Bio-Formats to version 6.10.0, see [release announcement]({{ site.baseurl }}/2022/05/31/bio-formats-6-10-0.html).
- the inclusion of the OMEZarrReader version 0.3.0 for reading OME-NGFF data

Note the Bio-Formats Memoizer cache **will be invalidated** on upgrade from earlier OMERO.server versions.

The loading of OME-NGFF data (version 0.1 to 0.4) is currently only available using the command-line

> interface - see the [limitations](https://omero.readthedocs.io/en/stable/sysadmins/limitations.html) page

Reviewer: ```suggestion
interface - see the [limitations](https://omero.readthedocs.io/en/stable/sysadmins/limitations.html#import-of-ome-ngff) page
```

Author: Or is it where we should use the reference that you dutifully introduced in https://github.com/ome/omero-documentation/pull/2248/files#r906087930?

Reviewer: Not sure how to technically do it. The  :ref:`minmax_limitation` is used, but from within the ome-docs repo. How to cite from the www repo ?

Reviewer: maybe intersphinx https://docs.readthedocs.io/en/stable/guides/intersphinx.html ?

Author: I think you can use it just like any heading anchor i.e. https://omero.readthedocs.io/en/stable/sysadmins/limitations.html#ngff-limitations in that case

Reviewer: Aha, thank you, updated the commit suggestion @sbesson

for more details.

Finally, the OMERO.server technical documentation has  been migrated to Read the Docs and

> the documentation for the stable version is available from https://omero.readthedocs.io/en/stable/.

Reviewer: ```suggestion
the documentation for the stable version is available from <https://omero.readthedocs.io/en/stable/>.
```

### Installing the Software:

> For full details of the changes with the OMERO 5.6 series see the

Reviewer: ```suggestion
For full details of the changes included in the OMERO 5.6 series see the
```

[OMERO 5.6.0]({{ site.baseurl }}/2020/01/15/omero-5-6-0.html) release
announcement. Full documentation for this release is available
under <https://omero.readthedocs.io/en/stable/>.

This release has been tested with
[OMERO.py 5.11.2](https://pypi.org/project/omero-py/5.11.2/) and

> [OMERO.web 5.14.1](https://pypi.org/project/omero-web/5.14.1/) so we

Reviewer: ```suggestion
[OMERO.web 5.14.1](https://pypi.org/project/omero-web/5.14.1/). We
```

> recommend that you upgrade OMERO.py and OMERO.web on your deployments.

Reviewer: ```suggestion
recommend that you upgrade OMERO.py and OMERO.web accordingly on your deployments.
```

---
layout: post
title: Release of OMERO.server 5.6.5
---

Today we are releasing OMERO.server 5.6.5 which includes:

-   a new server configuration allowing to control the pyramidal requirement for floating-point images
-   an upgrade of Bio-Formats to the last released version 6.10.0, see the [release announcement]({{ site.baseurl }}/2022/05/31/bio-formats-6-10-0.html).
-   the [OMEZarrReader](https://github.com/ome/ZarrReader/) version 0.3.0

The OMEZarrReader inclusion allows the server to import imaging data written according to the 
[OME-NGFF](https://ngff.openmicroscopy.org/) specification. The reader supports OME-NGFF datasets

> for all published versions of the specification, currently up to 0.4 and reads OME-XML

Reviewer: ```suggestion
for all published versions of the specification (up to and including 0.4) and reads OME-XML
```

metadata stored using to the layout written by Glencoe's

> [bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) utility and currently under

Reviewer: ```suggestion
[bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) utility. The OMEZarrReader is currently under
```

Reviewer: I am afraid you lost me there completely. I would suggest you reformulate the very long sentence yourself please, always stating the subject of the action in the sentence anew (not using ``it`` too much).  It seems that the complexity of the problem asks for at least one additional sentence.

> [bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) utility and currently under

Reviewer: ```suggestion
[bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) utility. These OME-XML metadata are currently under
```

> [bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) utility and currently under

Reviewer: @sbesson maybe like this ? The problem is the sudden change of focus in the sentence from the ZarrReader to ... OME-XML metadata ? (the other problem is that this sudden change of focus completely lost my understanding of the sentence)

> review for inclusion in the OME-NGFF specification - see [here](https://github.com/ome/ngff/pull/112).

Reviewer: ```suggestion
review for inclusion in the OME-NGFF specification - see [the relevant discussion on GitHub](https://github.com/ome/ngff/pull/112).
```

OME-NGFF import is currently only available using the command-line interface, see the
[limitations](https://omero.readthedocs.io/en/stable/sysadmins/limitations.html#ngff-limitations) page
for more details.

Note the Bio-Formats Memoizer cache **will be invalidated** on upgrade from earlier OMERO.server versions.

> Finally, the OMERO.server technical documentation has  been migrated to Read the Docs and

Reviewer: ```suggestion
Finally, the OMERO.server technical documentation has been migrated to Read the Docs and
```

Today we are releasing OMERO.server 5.6.5 which includes:

-   a new server configuration allowing to control the pyramidal requirement for floating-point images
-   an upgrade of Bio-Formats to the last released version 6.10.0, see the [release announcement]({{ site.baseurl }}/2022/05/31/bio-formats-6-10-0.html).
-   the [OMEZarrReader](https://github.com/ome/ZarrReader/) version 0.3.0

The OMEZarrReader inclusion allows the server to import imaging data written according to the 
[OME-NGFF](https://ngff.openmicroscopy.org/) specification. The reader supports OME-NGFF datasets
for all published versions of the specification, currently up to 0.4 and reads OME-XML

> metadata stored using to the layout written by Glencoe's

Reviewer 2: should be Glencoe Software
